models/marts/education/education_marts_tests.yml

```diff
@@ -1,14 +0,0 @@
-version: 2
-
-models:
-  - name: education_scholarship_categories_agg
-    description: "Aggregates scholarship categories for educational purposes."
-    columns:
```

Comment:
Are these test cases not valid any more ?

Reply:
Nope